### PR TITLE
Use  the native Promise for the server side code (close #3164)

### LINKED ISSUE #3163: Change Babel preset
- [ ] Enable native `async` functions

### LINKED ISSUE #3164: Use the native Promise in server code

### PR DESCRIPTION
**Pull request by miherlosev**
@AndreyBelym

### CODE REVIEW

File: src/utils/promisify.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import pify from 'pify';
```
Comment (AndreyBelym) [inline src/utils/promisify.js]: We can replace it with `util.promisify` (available in Node>=8.0.0) as well.

**Comment by AndreyBelym** [comment]
We can't use native Promises in `src/api/test-controller.js and `src/utils/re-executable-promise.js` until we improve the test run detection and close https://github.com/DevExpress/testcafe/issues/4139.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@AndreyBelym FPR

**Pull request merged**